tests/waterdyn4/database/definitions/scatterfactors-xray-table.xml

```diff
@@ -2,585 +2,925 @@
-			<param>1.4495	</param>
-			<param>64.7016	</param>
-			<param>0.0		 </param>
+			<param> 	14.0743	</param>
+			<param>3.26550	</param>
+			<param>7.03180	</param>
+			<param>0.233300	</param>
+			<param>5.16520	</param>
+			<param>10.3163	</param>
+			<param>2.41000	</param>
+			<param>58.7097	</param>
+			<param>1.30410	</param>
 		</element> 
 		<element>
 		<!-- ION Zn(2+) -->
 			<name>zinc</name>
 		 	<type>2</type>
-			<param>0.252	</param>
-			<param>0.600	</param>
-			<param>0.917	</param>
-			<param>0.663	</param>
-			<param>0.161	</param>
-			<param>0.161	</param>
-			<param>1.01	</param>
-			<param>2.76	</param>
-			<param>7.08	</param>
-			<param>19.0	</param>
-			<param>0.0		 </param>
+			<param>11.9719	</param>
+			<param>2.99460	</param>
+			<param>7.38620	</param>
+			<param>0.203100	</param>
+			<param>6.46680	</param>
+			<param>7.08260	</param>
+			<param>1.39400	</param>
+			<param>18.0995	</param>
+			<param>0.780700	</param>
 		</element> 
 
 		<element>
 			<name>Ga			</name>
 		 	<type>2</type>
-			<param> 	0.2135	</param>
-			<param>0.102	</param>
-			<param>0.9768	</param>
-			<param>1.0219	</param>
-			<param>1.6669	</param>
-			<param>4.6275	</param>
-			<param>2.5662	</param>
-			<param>22.8742	</param>
-			<param>1.679	</param>
-			<param>80.1535	</param>
-			<param>0.0		 </param>
+			<param> 	15.2354	</param>
+			<param>3.06690	</param>
+			<param>6.70060	</param>
+			<param>0.241200	</param>
+			<param>4.35910	</param>
+			<param>10.7805	</param>
+			<param>2.96230	</param>
+			<param>61.4135	</param>
+			<param>1.71890	</param>
+		</element> 
+		<element>
+			<name>Ga(3+)			</name>
+		 	<type>2</type>
+			<param> 	12.6920	</param>
+			<param>2.81262	</param>
+			<param>6.69883	</param>
+			<param>0.227890	</param>
+			<param>6.06692	</param>
+			<param>6.36441	</param>
+			<param>1.00660	</param>
+			<param>14.4122	</param>
+			<param>1.53545	</param>
 		</element> 
 		<element>
 			<name>Ge			</name>
 		 	<type>2</type>
-			<param> 	0.2135	</param>
-			<param>0.0989	</param>
-			<param>0.9761	</param>
-			<param>0.9845	</param>
-			<param>1.6555	</param>
-			<param>4.5527	</param>
-			<param>2.8938	</param>
-			<param>21.5563	</param>
-			<param>1.6356	</param>
-			<param>70.3903	</param>
-			<param>0.0		 </param>
+			<param> 	16.0816	</param>
+			<param>2.85090	</param>
+			<param>6.37470	</param>
+			<param>0.251600	</param>
+			<param>3.70680	</param>
+			<param>11.4468	</param>
+			<param>3.68300	</param>
+			<param>54.7625	</param>
+			<param>2.13130	</param>
 		</element> 
 		<element>
-			<name>As			</name>
+			<name>Ge(4+)			</name>
+		 	<type>2</type>
+			<param> 	12.9172	</param>
+			<param>2.53718	</param>
+			<param>6.70003	</param>
+			<param>0.205855	</param>
+			<param>6.06791	</param>
+			<param>5.47913	</param>
+			<param>0.859041	</param>
+			<param>11.6030	</param>
+			<param>0.393974	</param>
+		</element> 
+		<element>
+			<name>As	 <!-- wrong data  -->		</name>
```

Comment:
What is the meaning of `wrong data` here? In the PDF you linked to there's data for `As` (page 573) so why not include that data here?